packages/headless/src/controllers/search-box/headless-search-box.ts

```diff
@@ -164,5 +191,6 @@ function getSuggestions(state: QuerySuggestState | undefined) {
 
   return state.completions.map((completion) => ({
     value: completion.expression,
+    highlighted: completion.highlighted,
```

Comment:
Following on the comment above, let's remove this from the returned object, and adjust the method above to retrieve it directly.